src/components/Tipseen/TipseenBasicContent.tsx

```diff
@@ -1,11 +1,18 @@
 import cx from "classnames";
-import PropTypes from "prop-types";
 import TipseenTitle from "./TipseenTitle";
+import { VibeComponentProps } from "../../types";
+import { FC } from "react";
+import { ElementContent } from "../../types/ElementContent";
 import styles from "./TipseenBasicContent.module.scss";
 
 const BASE_CSS_CLASS = "monday-style-tipseen-basic-content";
 
-const TipseenBasicContent = ({ title, children, className }) => {
+interface TipseenBasicContentProps extends VibeComponentProps {
+  title?: string;
+  children?: ElementContent | ElementContent[];
```

Comment:
ElementContent is not enough?

Reply:
No cause in this case e.g. it's recieving an array of `ReactElement`s
<img width="993" alt="image" src="https://user-images.githubusercontent.com/104433616/212080100-8aba4fc8-1617-4e7d-8d26-521ee27d7352.png">